modules/meshtastic.nix

```diff
@@ -1,4 +1,4 @@
-{ config, lib, pkgs, ...}:
+self: { config, lib, pkgs, ...}:
```

Comment:
Making a module a function like isn't generally considered good practice, as I remember, as it breaks composability. There're however other ways to bring the flake's `self` into the scope via the tuple we already have.